hwdb.d/60-evdev.hwdb

```diff
@@ -399,6 +399,14 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# Lenovo X240 series
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPadX240:*
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPadX240?:*
```

Comment:
Hmm, the line above would also match. It was added in bbe7cdeb8fa6e880a1296e170648e468036e843b.

Reply:
Indeed. I should have said it but I asked first to Peter Hutterer and he said that will be ok
https://gitlab.freedesktop.org/libinput/libinput/-/issues/521

Reply:
I'll merge this with an adjusted commit message.